bigtable/integration_test.go

```diff
@@ -1272,16 +1274,20 @@ func TestIntegration_AdminCreateInstance(t *testing.T) {
 
 	// Basic return values are tested elsewhere, check instance type
 	if iInfo.InstanceType != DEVELOPMENT {
-		t.Fatalf("Instance is not DEVELOPMENT: %v", err)
+		t.Fatalf("Instance is not DEVELOPMENT: %v", iInfo.InstanceType)
+	}
+	if got, want := iInfo.Labels, conf.Labels; !reflect.DeepEqual(got, want) {
```

Comment:
Can we use `cmp.Equal` or `cmp.Diff` instead of `reflect. DeepEqual` throughout.